CHANGES.md

```diff
@@ -114,6 +114,10 @@ OpenSSL 3.5
 
    *Juhász Péter*
 
+ * Enabled x86-64 SM3 optimizations with SM3 ISA Extension.
+
+   *Alina Elizarova*
+
```

Comment:
could you add a comment here illustrating to users what sort of performance benefit they might expect, and in what use cases please?

Reply:
that looks good thank you